mcpwm_foc.c

```diff
@@ -4127,8 +4154,8 @@ static void svm(float alpha, float beta, uint32_t PWMHalfPeriod,
 	// sector 3-4
 	case 3: {
 		// Vector on-times
-		uint32_t t3 = (TWO_BY_SQRT3 * beta) * PWMHalfPeriod;
-		uint32_t t4 = (-alpha - ONE_BY_SQRT3 * beta) * PWMHalfPeriod;
+		uint32_t t3 = 2.0 * beta13 * PWMHalfPeriod;
```

Comment:
This could also be left as `TWO_BY_SQRT3`. The reason to use `2.0 * beta13` is to let the compiler know we don't need `beta` anymore. This isn't a very strong reason and if it's found that there's a readability preference for the old format then let's go back to that.